main/zoe/api/zoe-helpers.md

```diff
@@ -173,9 +173,44 @@ import { makeZoeHelpers } from '@agoric/zoe/src/contractSupport/zoeHelpers';
 
 const { swap } = makeZoeHelpers(zoe);
 
-  const seat = harden({
-    matchOffer: () => swap(firstInviteHandle, inviteHandle),
-  });
+  // `firstOfferHandle` is from a prior offer to the contract
+  const hook = newHandle => swap(firstInviteHandle, newHandle);
```

Comment:
We should change any mention of inviteHandle to offerHandle

Reply:
Done